### DISCUSSION TITLE
[Suggestion] A way of using both all the DNSCypt relays and all the ODOH relays.

### DISCUSSION THREAD

**Discussion by phantomcraft** [DNSCrypt/dnscrypt-proxy#1869] [Ideas]
Let's suppose someone wants to use both all the DNSCrypt servers through relays and all the ODOH servers/relays at the same time. It's not possible to do that because wildcard * only selects one type of relay at the same time, not both of them (ODOH and DNSCrypt relays).

It could be like this:

```
routes = [
    { server_name='*dnscrypt*', via=['dnscrypt-relay-1', 'dnscrypt-relay-2'] }
]

routes = [
    { server_name='*odoh*', via=['odoh-relay-1', 'odoh-relay-2'] }
]

skip_incompatible = true
```

**Comment by jedisct1** [comment]
It is already possible. A wildcard list can include both DNSCrypt and ODoH relays. The proxy automatically selects the relevant subset for each protocol.

**Comment by phantomcraft** [comment]
Ok, I didn't know that were possible.
